iset.mm

```diff
@@ -86216,6 +86216,38 @@ would be equivalent to negated equality and this would follow readily (for
     wi df-stab sylibr ) ACDBCDEZABFZGZGZUBRUBHUAUDABIJZGUBUAUEUCUAUEABKUCABLABM
     NOABPQUBST $.
 
+  ${
+    $d A r s t u x y $.  $d B r s t u x y $.
+    $( Reverse closure for apartness.  (Contributed by Jim Kingdon,
+       19-Dec-2023.) $)
+    aprcl $p |- ( A =//= B -> ( A e. CC /\ B e. CC ) ) $=
+      ( vr vs vt vu vx vy cap cv ci co wceq wa cr wrex cc wcel 2rexbidv cvv wbr
+      cop c1st cmul caddc c2nd creap wo df-br copab eqeq1 anbi1d anbi2d elopabi
+      cfv df-ap eleq2s reximi syl relopabi brrelex1i ad3antrrr brrelex2i op1stg
+      sylbi simpl syl2anc simprl ad2antrr recnd ax-icn a1i simprr mulcld addcld
+      eqeltrd eqeltrrd op2ndg recn adantr adantl jca ex rexlimdvva mpd ) ABIUAZ
+      ABUBZUCUOZCJZKDJZUDLZUELZMZWGUFUOZEJZKFJZUDLZUELZMZNZFOPZEOPZDOPZCOPZAQRZ
+      BQRZNZWFWTWIWOUGUAWJWPUGUAUHZNZFOPZEOPZDOPZCOPZXDWFWGIRXMABIUIXMWGGJZWLMZ
+      HJZWRMZNZXHNZFOPEOPZDOPCOPZGHUJIYAWMXQNZXHNZFOPEOPZDOPCOPXMGHWGXNWHMZXTYD
+      CDOOYEXSYCEFOOYEXRYBXHYEXOWMXQXNWHWLUKULULSSXPWNMZYDXKCDOOYFYCXIEFOOYFYBW
+      TXHYFXQWSWMXPWNWRUKUMULSSUNGHFEDCUPZUQVEXLXCCOXKXBDOXJXAEOXIWTFOWTXHVFURU
+      RURURUSWFXBXGCDOOWFWIORZWJORZNZNZWTXGEFOOYKWOORZWPORZNZNZWTXGYOWTNZXEXFYP
+      WHAQYPATRZBTRZWHAMWFYQYJYNWTABIYAGHIYGUTZVAVBZWFYRYJYNWTABIYSVCVBZABTTVDV
+      GYPWHWLQYOWMWSVHYPWIWKYPWIYKYHYNWTWFYHYIVHVIVJYPKWJKQRZYPVKVLYPWJYKYIYNWT
+      WFYHYIVMVIVJVNVOVPVQYPWNBQYPYQYRWNBMYTUUAABTTVRVGYPWNWRQYOWMWSVMYOWRQRZWT
+      YNUUCYKYNWOWQYLWOQRYMWOVSVTYNKWPUUBYNVKVLYMWPQRYLWPVSWAVNVOWAVTVPVQWBWCWD
+      WDWE $.
+  $}
+
+  ${
+    $d A x y $.  $d B x y $.
+    $( The points apart from a given point are complex numbers.  (Contributed
+       by Jim Kingdon, 19-Dec-2023.) $)
+    apsscn $p |- { x e. A | x =//= B } C_ CC $=
```

Comment:
Do we need `x e. A`? I think already `{ x | x =//= B } C_ CC` would be true.

Reply:
Yes `{ x | x =//= B } C_ CC` would also be provable but `{ x e. A | x =//= B }` is the one we are likely to use more. This is in the literature using names like "apartmess space" and sometimes has its own notation.